src/kOS/UserIO/TelnetMainServer.cs

```diff
@@ -57,7 +57,6 @@ public TelnetMainServer()
             
             DontDestroyOnLoad(transform.gameObject); // Otherwise Unity will stop calling my Update() on the next scene change because my gameObject went away.
 
-            Console.WriteLine("kOS TelnetMainServer class exists."); // Console.Writeline used because this occurs before kSP's logger is set up.
```

Comment:
caught this when I did a scan through all the code for cases where there was a WriteLine, to see if any of them were leftover debug crud, and this one was.